src/nord.sh

```diff
@@ -266,13 +266,14 @@ print_help() {
 # @return 0 if all required dependencies are validated, 1 otherwise
 # @since 0.2.0
 validate_dependencies() {
-  declare -a missing_deps deps=("${!1}")
+  declare -a deps=("${!1}")
+  declare -a missing_deps=()
   for exec in "${deps[@]}"; do
     if ! command -v "${exec}" > /dev/null 2>&1; then
       missing_deps+=(${exec})
     fi
   done
-  if [ ${#missing_deps[*]} -eq 0 ]; then
+  if [ "${#missing_deps[*]}" -eq 0 ]; then
```

Comment:
This can lead to problems for the iterator values so let's revert this.

```suggestion
  if [ ${#missing_deps[*]} -eq 0 ]; then
```